gravitee-am-ui/src/app/domain/settings/saml2/saml2.component.ts

```diff
@@ -55,11 +59,11 @@ export class Saml2Component implements OnInit {
       this.publicKeys(this.domainSamlSettings.certificate);
     }
 
-    this.pluginMetadata = {'deployed': false, 'feature': 'am-idp-saml2'};
     const samlPlugin = this.route.snapshot.data['identities']['am-idp-saml'];
     if (samlPlugin != null) {
-      this.pluginMetadata.deployed = samlPlugin.deployed;
+      this.saml2Feature.deployed = samlPlugin.deployed;
     }
+    this.isMissingSaml2Feature$ = this.licenseService.isMissingFeature$(this.saml2Feature);
```

Comment:
I would rather than use a resolver to get the license details in order to check if the plugin is deployed or not